packages/yew/src/html/html_ref.rs

```diff
@@ -0,0 +1,391 @@
+use std::any::Any;
+use std::cell::{Ref, RefCell};
+use std::fmt;
+use std::marker::PhantomData;
+use std::ops::Deref;
+use std::rc::Rc;
+
+use wasm_bindgen::JsValue;
+use web_sys::Node;
+
+use super::{AnyScope, Scope};
+use crate::BaseComponent;
+
+/// Trait for types that can be referenced with a [`HtmlRef`].
+pub trait ErasedStorage {
```

Comment:
Does this mean that for every type that wants be stored inside `HtmlRef`, it needs to implement this trait?

Reply:
What happens if for any `T` `T::Erased = Rc<dyn Any>`?

Reply:
If we could stop running away from ownership and lifetime issues by wrapping everything in an `Rc` that would be nice.